### PR TITLE
Refactorings and Renamings

### PR DESCRIPTION
**Pull request by robintibor**

### CODE REVIEW

**Comment by agramfort** [comment]
don't forget to updated https://github.com/braindecode/braindecode/blob/master/docs/api.rst with new public functions and classes. thx

**Comment by robintibor** [comment]
after finishing of refactoring I will make a whole pass/restructure doc starting page etc.

**Comment by robintibor** [comment]
> mikrovolt -> microvolt

ja nagut, aber eigentlich ist es mikrovolt ;p ;) but changed, yes.

File: braindecode/classifier.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from skorch.classifier import NeuralNetClassifier
 from skorch.utils import train_loss_score, valid_loss_score, noop
 
-from .scoring import PostEpochTrainScoring, CroppedTrialEpochScoring
+from braindecode.training.scoring import PostEpochTrainScoring, CroppedTrialEpochScoring
```
Comment (agramfort) [inline braindecode/classifier.py:8]: I think we only use full imports for tests. Here it should be a relative import

File: braindecode/datasets/base.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ class BaseDataset(Dataset):
     Parameters
     ----------
     raw: mne.io.Raw
-    description: dict | pandas.Series | None
+    description: dict | pandas.Series | Nonew
```
Comment (agramfort) [inline braindecode/datasets/base.py:27]: Nonew
->
None

**Comment by agramfort** [comment]
@robintibor I would advise to get at least one approval from someone else before merging.
It's a necessary step to have more people know the history of the code base and also spot typos I saw by glancing over the first files.

**Comment by robintibor** [comment]
yes now everything a bit rushed for presentation today, will try to keep to it again afterwards, and do things earlier next time. Thanks for catches

**Pull request merged**